cernopendata_client/cli.py

```diff
@@ -243,6 +243,12 @@ def get_file_locations(server, recid, doi, title, protocol, expand, verbose):
         DOWNLOAD_RETRY_SLEEP
     ),
 )
+@click.option(
+    "--download-engine",
```

Comment:
We are using `download-engine` but elsewhere in the code we have `downloader-engine`:

```
tests/test_cli_download_files.py
127:    """Test download_files() command with downloader-engine option."""
145:    """Test download_files() command with downloader-engine option and wrong protocol."""
157:    """Test download_files() command with downloader-engine option and wrong protocol."""
```

Let's standardise this.  

BTW what about the simplest version `downloader` so that people type `--downloader pycurl`?  Seems good to go nicely with class names...

Reply:
I was thinking about `downloader` initially but then I thought `download-engine` sounds to be more specific.